README.md

```diff
@@ -1600,7 +1600,28 @@ Finished in 0.08 seconds
 Randomized with seed 452478
 ```
 
+### 10 Use `.json` in url
 
+If you want to access the json data via the url,
+ you can add `.json` at the end of the your endpoint.
+
+`Content` will automatically recognise the request
+update the accept header to be `application/json`
+and send back the data as json.
+
+For that you just need to update the `router.ex` file 
+to define the `/.json` route:
+
+```elixir
+    get "/", QuotesController, :index
+    get "/.json", QuotesController, :index
```

Comment:
we should not _need_ to add a `get /.json` route. we should be able to use a wildcard.
(_otherwise we will need to duplicate all routes just to have this functionality..._)
we will need to revise the requirements in `content` to accommodate this.

Reply:
Fixed by: https://github.com/dwyl/content/pull/5 🚀